app.rb

```diff
@@ -1,12 +1,78 @@
+require 'aws-sdk-secretsmanager'
 require 'sinatra'
 require 'pry-byebug'
-require "./efiler_service.rb"
+require 'jwt'
+require 'dotenv/load'
+require 'nokogiri'
 
-get '/' do
-  'Hello world!'
+require_relative "efiler_service"
+require_relative "acks"
+require_relative "submissions_status"
+require_relative "extensions/efiler_api_client_auth"
+
+before do
+  content_type 'application/json'
+end
+
+authenticate_jwt
+
+post '/submit' do
+  unless params.has_key?("submission_bundle")
+    status 400
+    return { error_message: "Request missing submission_bundle param" }.to_json
+  end
+
+  submission_filename = params["submission_bundle"]["filename"]
```

Comment:
consider checking that filename exists in case request is malformed (not a file) to avoid NoMethodError

Reply:
done!